### PR TITLE
Include Fluxcal info in report

### PR DESCRIPTION
**Pull request by KimMcAlpine**
The report now provides a table of measured fluxes of gain calibrators as provided in
`telstate['measured flux']`. A model value is included in the table for comparison if a model is provided.

Additionally if a flux model exists for a gain calibrator it is now plotted in all plots of amplitude versus frequency.

In all plots of the gain calibrators the visibilities are scaled to the measured fluxes reported in telstate[`measured_flux`].

The target data is not gain calibrated, as the flux of the target is unlikely to be known `a priori` this doesn't really have
a negative impact on the diagnostic value of the target plots in the report, and avoids the need to introduce a complicated scheme
to correctly interpolate gains between different gain calibrators onto the target

### CODE REVIEW

File: katsdpcal/report.py [hunk 1]
```diff
@@ -784,6 +830,12 @@ def write_g_time(report, report_path, flux_cal, targets, av_corr,
                 av_data = [d * flux_cal[target_name] for d in av_data]
             data += av_data
             times += av_times
+            units.append(_get_units(kat_target, correlator_freq, flux_cal))
+
+        if all([u == 'Jy' for u in units]):
+            units = 'Jy'
+        else:
+            units = 'arb'
```
Comment (mauch) [inline katsdpcal/report.py]: I like 'arb' :)

File: katsdpcal/report.py [hunk 2]
```diff
@@ -177,6 +177,37 @@ def write_bullet_if_present(report, table, var_text, var_name, transform=None):
     report.writeln('* {0}:  {1}'.format(var_text, value))
 
 
+def _get_units(target, freqs, flux_cal={}):
+    """ Determine the amplitude units of a target
```
Comment (mauch) [inline katsdpcal/report.py]: The convention throughout the rest of cal (and in [PEP 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings)) seems to be to omit the space between the """ and the first character of the summary line, and also to end it with a full stop.

File: katsdpcal/report.py [hunk 3]
```diff
@@ -362,6 +393,75 @@ def write_table_timecol(report, antenna_names, times, data, ave=False):
     report.writeln()
 
 
+def write_flux_cal(report, flux_cal, flux_std, parameters, targets):
+    """ Write Flux calibration info in a table
```
Comment (mauch) [inline katsdpcal/report.py]: Here as well.

Comment (KimMcAlpine) [reply katsdpcal/report.py]: Done

File: katsdpcal/report.py (hunk 2 above)
Comment (KimMcAlpine) [reply katsdpcal/report.py]: Done

**Pull request merged**